app/src/main/java/ca/ualberta/CMPUT3012019T02/alexandria/model/message/ImageMessage.java

```diff
@@ -8,12 +8,12 @@ public ImageMessage(String content, String status, Date date, String sender) {
         super("image", content, status, date, sender);
     }
 
-    public String getImageUrl() {
+    public String getImage() {
         return getContent();
     }
 
-    public void setImageUrl(String imageUrl) {
-        setContent(imageUrl);
+    public void setImage(String imageId) {
```

Comment:
Could you please update tests to be image id also?

Reply:
Replace gs:// with something like 16bb3894-be95-4108-9b9d-c6f5283a7920 and rename method names

Reply:
Oops wrong account (I can aprove from sazzy4o later)

Reply:
I updated the tests